compiler/lib/php_from_rehp.re

```diff
@@ -102,7 +103,12 @@ let binop_from_rehp = binop =>
   };
 
 let empty = {names: StringMap.empty, vars: Code.Var.Map.empty};
-let emptyOutput = {dec: empty, use: empty, use_label: false};
+let emptyOutput = {
+  dec: empty,
+  use: empty,
+  use_label: false,
+  use_continue: false,
```

Comment:
I think that either renaming this to `continues_outside` or keeping the name but adding a comment would make it clear what this variable refers to. It tracks whether or not a part of the AST continues outside of that AST. Things can "use continue" inside of an AST that merely continue within that AST and that's not what this variable tracks.